backend/qemu.pm

```diff
@@ -614,6 +624,14 @@ sub _set_graphics_backend ($self, $is_arm) {
     sp('device', "${device}${options}");
 }
 
+sub get_qemu_version($self, $qemubin) {
```

Comment:
This is not a simple getter (as it actually invokes a sub process) so maybe `determine_…` would make more sense:
```suggestion
sub determine_qemu_version($self, $qemubin) {
```

Reply:
done